roles/puppet_repositories/tasks/puppetlabs-5-redhat.yml

```diff
@@ -2,6 +2,7 @@
 - name: 'Setup Puppet 5 Repository'
   yum:
     name: https://yum.puppet.com/puppet5-release-el-{{ ansible_distribution_major_version }}.noarch.rpm
```

Comment:
5 and 6 are now very similar. With Puppet 7 coming soon, I wonder if we can use variables

Reply:
Probably. But 6 is also purging 5, so it's more than one simple variable.

I'd defer that to a later point.

Reply:
Right. We do that because they conflict, but don't upgrade in place. I opened https://tickets.puppetlabs.com/browse/CPR-762